saleor/core/utils/random_data.py

```diff
@@ -47,10 +47,12 @@
 COLLECTIONS_SCHEMA = [
     {
         'name': 'Summer collection',
-        'image_name': 'summer.jpg'},
+        'image_name': 'summer.jpg',
+        'description': 'Description of a summer collection.'},
```

Comment:
Could you get a nicer copy for this? Right now populatedb creates real-like data